src/client/components/pages/parts/revisions-table.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2019 Prabal Singh
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import * as bootstrap from 'react-bootstrap';
+import * as utilsHelper from '../../../helpers/utils';
+import {genEntityIconHTMLElement, getEntityLabel} from '../../../helpers/entity';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {kebabCase as _kebabCase} from 'lodash';
+
+
+const {Table} = bootstrap;
+const {formatDate} = utilsHelper;
+
+
+function RevisionsTable(props) {
+	const {results} = props;
+	const tableCssClasses = 'table table-striped';
+	return (
+
+		<div>
+			<div>
+				<h1 className="text-center">Recent Activity</h1>
+			</div>
+			<hr className="thin"/>
+			<Table
+				responsive
+				className={tableCssClasses}
+			>
+				<thead>
+					<tr>
+						<th className="col-sm-2">Revision ID</th>
+						<th className="col-sm-5">Revision</th>
+						<th className="col-sm-3">Edited By</th>
+						<th className="col-sm-2">Date</th>
+
+					</tr>
+				</thead>
+				<tbody>
+					{
+						results.map((revision, i) => (
+							<tr key={i}>
+								<td>
+									<a href={`/revision/${revision.revisionId}`} >
+										{revision.revisionId}
+									</a>
+								</td>
+								<td>
+									{revision.entities.map(entity => (
+										<div key={`${revision.revisionId}-${entity.bbid}`}>
+											<a href={`/${_kebabCase(entity.type)}/${entity.bbid}`} >
```

Comment:
Actually, here you can use the `getEntityLink` utility (from src/server/helpers/utils) to remove some code duplication